tests/test_core.py

```diff
@@ -137,6 +138,14 @@ class TestComponents:
         "name": {"type": "string", "example": "doggie"},
     }
 
+    @pytest.mark.parametrize("spec", ("3.0.0",), indirect=True)
+    def test_example(self, spec):
+        # Referenced examples are only supported in OAS 3.x
```

Comment:
[Nitpick] I'd put this comment above `@pytest.mark.parametrize("spec", ("3.0.0",), indirect=True)`

Reply:
Fixed in 0776441f5060f10767cdd773fc5961c252a024c0